### PR TITLE
Add cohort calendar instructions and photo

### PR DESCRIPTION
**Pull request by ErikElliott-ynp**
I've added documentation for how students can get to their cohort's calendar. Let me know if you want to change any of the verbiage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +37,13 @@ Here you will find all resources which will be given out throughout your time he
 * Golden Gate Instructors 
     * golden-gate-instructors@appacademy.io
 
+## Cohort Calendar
+* Your Cohort's Calendar is found on the README of your Cohort's Branch
+* To get there, go to the top of the page and click the branch dropdown
```
Comment (carlosaicrag) [inline README.md:42]: "to get to your branch, go to the top of the page, and click the branch dropdown. This process is demonstrated in the image below". Maybe we can reword it to be this so that we explicitly say what the image is demonstrating.

Comment (carlosaicrag) [inline README.md:42]: can we replace the bullet points with numbered steps.

File: README.md [hunk 2]
```diff
@@ -36,6 +37,14 @@ Here you will find all resources which will be given out throughout your time he
 * Golden Gate Instructors 
     * golden-gate-instructors@appacademy.io
 
+## Cohort Calendar
+**Your Cohort's Calendar is found on the README of your Cohort's Branch**
+1. To get to your branch, go to the top of the page and click the branch dropdown
+2. Then, click on the branch with the name of your Cohort's start date
+    + This process is illustrated below
+ ![calendar](./images/calendar-branch.png)
+3. The Calendar link will then be on the README
```
Comment (carlosaicrag) [inline README.md:46]: What do you think about adding one more picture here that shows what the calendar link looks like in the README.  Do you think it would be more helpful to add the picture or do you think it's fine with just the third instruction? If you think it would be more helpful you should go ahead and add it but if you think it's fine then I can go ahead and merge it now.  Just let me know.

Comment (ErikElliott-ynp) [reply README.md:46]: Yeah I was thinking about that but it's the only thing on the README and pretty in your face once you click the link. I can add one if we want tho

Comment (carlosaicrag) [reply README.md:46]: I guess just to be safe we should add a picture. If you could add one that would be great.  Thank you in advance.

Comment (ErikElliott-ynp) [reply README.md]: will do!
